flow-sample-basic/src/main/java/flow/sample/basic/HelloScreen.java

```diff
@@ -0,0 +1,47 @@
+package flow.sample.basic;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+final class HelloScreen implements Parcelable {
```

Comment:
Actually debating that whole saving-people-from-Parcelable thing.  Need to do some perf analysis.